@navikt/ds-css/baseline/index.css

```diff
@@ -0,0 +1,16 @@
+@import "~normalize.css";
+@import "~@navikt/ds-tokens/dist/tokens.css";
+@import "Source-Sans-Pro-regular.css";
+@import "Source-Sans-Pro-italic.css";
+@import "Source-Sans-Pro-600.css";
+
+body {
+  font-family: var(--navds-font-family);
+  line-height: 1.375;
+  color: var(--navds-color-text-primary);
+  font-size: 1.125rem;
+}
+
+a {
```

Comment:
Burde denne egentlig være med? Jeg tenker jo vi helst vil at folk bruker link-komponenten, men det gir kanskje mening å ha default styles som gir mening som fallback? Burde vi i så fall lage flere grunnstiler som gir mening? F.eks. h1-h6, input, button, etc.

Reply:
Hmm, vi burde kanskje gå gjennom nav-frontend-core å se gjennom hva vi ønsker å ta med oss før vi går live. Er mye diverse løsninger som fikser småting, samt styles for print etc

Reply:
Ja, kan ta en runde gjennom de gamle filene på det. Men jeg stemmer for at vi tar med minst mulig. Jeg tror det er lurt å fjerne denne `a`-stilen også, og heller legge den til igjen om det mot formodning skulle dukke opp et ønske om det.

Reply:
Jeg gikk gjennom core.less nå, og jeg fant egentlig ingenting jeg tror er kritisk, annet enn muligens `.sr-only` og eventuelt en print.css. Hva tenker du om de?

Reply:
Tror `.sr-only` og print er greit å ta med videre da de løser reelle problemer for eks skjemlesere og utskrift av infosider/søknaer

Reply:
Jeg ser at alle lenkene i `print.less` er døde, men jeg fant den samme fila som de kopierte fra i 2017 fortsatt er nesten lik (https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/style.css). Skal vi bare kopiere den nyeste versjonen, eller skal vi prøve å spikke noe helt nytt?

Reply:
Kan kanskje bare kopiere den, da å lage en ny vil ta litt tid regner jeg med.

Reply:
https://github.com/navikt/nav-frontend-moduler/pull/920